lunes_cms/help/templates/public_upload.html

```diff
@@ -65,28 +65,46 @@
         3: "das",
         4: "die (Plural)"
       }
+      var buffer_array = []
 
-      $( document ).ready(function() {
+      function Comparator(a, b) {
+        if (a[1] < b[1]) return -1;
+        if (a[1] > b[1]) return 1;
+        return 0;
+      }
 
+      $( document ).ready(function() {
+        $("#inputDiscipline").append("<option value='0'>Alle Berufe</option>")
```

Comment:
I still think that this static option should be placed in the html source code instead of JS.
Adding an option dynamically via JS does only make sense when it should only be visible under certain conditions, but in this case, the option is just visible all the time.

Reply:
Oh, I'm sorry. I must have mixed it up. Now it should be clean, inside the html code